src-js/fontra-core/src/glyph-data.js

```diff
@@ -111,3 +116,48 @@ export function getGlyphInfoFromGlyphName(glyphName) {
   parseGlyphDataCSV();
   return glyphDataByName.get(glyphName);
 }
+
+export function guessGlyphPlaceholderString(codePoints, glyphName) {
+  let glyphString = "";
+  if (codePoints?.[0]) {
+    glyphString = getCharFromCodePoint(codePoints[0]);
+  }
+
+  if (!glyphString && glyphName) {
+    const [baseGlyphName, extension] = splitGlyphNameExtension(glyphName);
+
+    let baseGlyphNames = [baseGlyphName];
+    if (baseGlyphName.indexOf("_") != -1) {
+      const [base, suffix] = splitGlyphNameExtension(baseGlyphName, "-");
+      baseGlyphNames = base.split("_").map((name) => name + suffix);
+    }
+
+    const codePoints = baseGlyphNames.map((name) => getCodePointFromGlyphName(name));
+    if (codePoints.length == baseGlyphNames.length) {
```

Comment:
Isn't this condition always true? Or do you mean to filter out null/undefined from the `codePoints` array?